src/ape_test/providers.py

```diff
@@ -1,8 +1,94 @@
-from typing import Iterator
-
+from typing import Any, Iterator
+
+from eth_keys import KeyAPI  # type: ignore
+from eth_tester.backends import PyEVMBackend  # type: ignore
+from eth_tester.backends.pyevm.main import (  # type: ignore
+    generate_genesis_state_for_keys,
+    get_default_genesis_params,
+)
+from eth_tester.exceptions import TransactionFailed  # type: ignore
+from eth_utils import to_wei
+from eth_utils.exceptions import ValidationError
+from eth_utils.toolz import assoc
+from hexbytes import HexBytes
 from web3 import EthereumTesterProvider, Web3
 
-from ape.api import ProviderAPI, ReceiptAPI, TransactionAPI
+from ape.api import ProviderAPI, ReceiptAPI, TransactionAPI, TransactionStatusEnum
+from ape.exceptions import ContractLogicError, OutOfGasError, VirtualMachineError
+from ape.utils import generate_dev_accounts
+
+
+def _setup_tester_chain(
```

Comment:
This part can be removed and this module can be a lot simpler once https://github.com/ethereum/eth-tester/pull/209 merges and gets released